src/main/java/teammates/logic/api/Logic.java

```diff
@@ -2488,6 +2493,17 @@ public void sendCommentNotification(String courseId) {
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, courseId);
         commentsLogic.sendCommentNotification(courseId);
     }
+    
+    /**
+     * This method sends all links (sent previously in the course) to
+     * the new email address.
+     */
```

Comment:
Move this documentation to the method in `feedbackSessionsLogic` instead. Also, some changes possible:
- Remove ~~This method~~
- Sends "all links": what is "all links"? Can be more specific here.
- Add `@param` documentation for all parameters used.
- This method (and the one in `FeedbackSessionsLogic` does not look like it need to return anything. If that is the case, make it a `void` method.